popup/popup.js

```diff
@@ -39,11 +39,12 @@ chrome.storage.local.get(["websiteUrl"]).then(({ websiteUrl }) => {
 			document.querySelector("#session-image").src = user.image;
 			document.querySelector("#session-name").innerHTML = user.name;
 			document.querySelector("#session-email").innerHTML = user.email;
-
+			const cookie = await chrome.cookies.get({ url: websiteUrl, name: '__Secure-next-auth.session-token' });
```

Comment:
Please add a catch clause here. We don't want to break the extension for all other use cases.